gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
brainfart using `-eq` instead of `-ne`
`-eq "2"` seems fine until you consider the case of sort not being available all together on the machine. which would give 127. Oh the pleasures of coding in bash. 

Maybe is should do a `check_command_exists sort` along with `sort -V >/dev/null 2>&1 `check